docs/resources/scenarios/task-reminders.md

```diff
@@ -82,9 +80,9 @@ This scenario uses [Power Automate](https://flow.microsoft.com) and [Microsoft T
     }
     ```
 
-5. Save the script with the name **Get People**.
+1. Save the script with the name **Get People**.
 
-6. Next, we need a second script to process the status report cards and put the new information in the spreadsheet. In the **Code Editor** task pane, press **New Script** and paste the following script into the editor.
+1. Next, we need a second script to process the status report cards and put the new information in the spreadsheet. In the **Code Editor** task pane, select **New Script** and paste the following script into the editor.
```

Comment:
In my limited experience so far, it seems like Code Editor is usually regular text and not bold.  
```suggestion
1. Next, we need a second script to process the status report cards and put the new information in the spreadsheet. In the Code Editor task pane, select **New Script** and paste the following script into the editor.
```